setup.py

```diff
@@ -85,13 +85,15 @@ def get_git_sha():
         "idna",
         "isodate",
         "markdown>=3.0",
+        "msgpack>=0.6.1, <0.7.0",
```

Comment:
I checked dependency tree for rogue licenses (even though FOSSA should catch if any) and LGTM